docs/src/pages/[platform]/getting-started/accessibility/examples/AriaLiveExample.tsx

```diff
@@ -0,0 +1,19 @@
+import { Flex, Button, Badge } from '@aws-amplify/ui-react';
+import { useState } from 'react';
+
+export const AriaLiveExample = () => {
+  const [count, setCount] = useState(0);
+
+  function handleClick() {
+    setCount(count + 1);
+  }
+
+  return (
+    <Flex>
+      <Button onClick={handleClick}>Add card to list.</Button>
+      <Badge aria-live="polite" alignSelf="center">
+        {`${count} cards in list`}
```

Comment:
Voiceover only announces the {count} and not the whole string unless we use a string literal here  which is neat and fun. 😒 Something about it being separated into different text leafs in the accessibility tree if we just use `{count} cards in list`.

Reply:
Cool example!

Reply:
Wow, good to know. Using `aria-atomic` is wired in this case